corescore/models.py

```diff
@@ -32,22 +32,32 @@ def __init__(self, path, batch_size=1, wd=1e-2,
         self.wd = wd
         self.pct_start = pct_start
         self.epochs = epochs
-        self.reduce_size = np.array([148, 1048])  # TODO derive from samples e.g. src_size / 6 # noqa: E501
+#        self.reduce_size = np.array([148, 1048])  # TODO derive from samples e.g. src_size / 6 # noqa: E501
+
+    def image_resize(self, sample, resize=4):
+        """Accepts a sample image from our training set.
+        Returns the dimensions, resized by 1/resize
+        (as a numpy array, to stay close to the original"""
+        # input dimensions are a torch.Size
+        size = sample.size
+        return np.array([int(size[0]/resize), int(size[1]/resize)])
 
     def image_src(self):
         """Load images from self.path/images"""
-        self.src = (
-            SegmentationItemList.from_folder(
-                self.path_img).split_by_rand_pct().label_from_func(
-                    self.get_y_fn, classes=LABELS))  # noqa: E501
+        self.src = (SegmentationItemList.from_folder(self.path_img).split_by_rand_pct().label_from_func(self.get_y_fn, classes=LABELS))  # noqa: E501
         self.src.train.y.create_func = partial(open_mask, div=True)
         self.src.valid.y.create_func = partial(open_mask, div=True)
         return self.src
 
-    def image_data(self):
+    def image_data(self, resize=4):
         """Load scaled-down source images as data"""
-        self.data = self.image_src().transform(
-            get_transforms(), size=self.reduce_size, tfm_y=True).databunch(
+        self.data = self.image_src()
+
+        # Resize (based on dimensions of first sample
+        resize_to = self.image_resize(self.data.x[0], resize=resize)
```

Comment:
Ok the issue must be here, detaching the chaining of operations is not having the result I expect, and this is what is causing the LabelList to lose properties it expects. Better to read dimensions from a sample input image OR to pass them in at model creation (obviating most of this code!)

Reply:
This just needed to be assigned rather than done in-place thankfully